modules/reactor/doc/content/source/meshgenerators/CoreMeshGenerator.md

```diff
@@ -24,6 +24,27 @@ The `CoreMeshGenerator` objects automatically assigns boundary information. The
 
 If the core is extruded to three dimensions the top-most boundary ID must be assigned using [!param](/Mesh/ReactorMeshParams/top_boundary_id) and will have the name "top", while the bottom-most boundary must be assigned using [!param](/Mesh/ReactorMeshParams/bottom_boundary_id) and will have the name "bottom".
 
+## Metadata Information
+
+Users may be interested in defining additional metadata to represent the reactor geometry and region IDs assigned to each geometry zone, which may be useful to users who want mesh geometry and composition information without having to inspect the generated mesh itself. In order to do so,  [ReactorMeshParams](ReactorMeshParams.md)/[!param](/Mesh/ReactorMeshParams/generate_rgmb_metadata) must be set to true. [!param](/Mesh/CoreMeshGenerator/show_rgmb_metadata) can be set to true in order to see the values of these metadata entries as console output.
+
+At the core level, each metadata entry is prepended with the prefix `"core_"`, and the following metadata is defined on the assembly mesh:
+
+- `mesh_dimensions`: Number of dimensions in core mesh, equivalent to  [ReactorMeshParams](ReactorMeshParams.md)/[!param](/Mesh/ReactorMeshParams/dim)
```

Comment:
Isn't this confusing since it implies that mesh_dimensions is an input parameter and not a metadata name? Also I don't think this would work since `dim` is the parameter name, not `mesh_dimensions`

Reply:
Yeah I guess we don’t have a way to make sure the metadata names don’t change

Reply:
Keeping this as is, just because the variable name (dim) is not as descriptive as the metadata name (mesh_dimensions)